modules/services/fb_streetview_ai_suggestions.js

```diff
@@ -0,0 +1,499 @@
+import _forEach from 'lodash-es/forEach';
+
+import { dispatch as d3_dispatch } from 'd3-dispatch';
+import { xml as d3_xml } from 'd3-fetch';
+
+import { coreGraph, coreTree } from '../core';
+import { osmEntity, osmNode, osmWay, osmRelation } from '../osm';
+import { utilRebind, utilStringQs, utilTiler } from '../util';
+
+// constants
+var APIROOT = 'https://www.gaoxm.sb.facebook.com/maps/ml_roads';
```

Comment:
Shouldn't this be the facebook.com endpoint?

Reply:
rather, 'mapwith.ai'

Reply:
yes, will update once the backend diff for fixing dummy data is pushed.